docs/sections/user_guide/cli/drivers/chgres_cube.rst

```diff
@@ -41,7 +41,7 @@ All tasks take the same arguments. For example:
 
    Required arguments:
      --config-file PATH, -c PATH
-         Path to config file
+         Path to UW YAML config file
```

Comment:
Here and in similar contexts, add `UW YAML` to make the intent/requirement clearer.